packages/standalone/src/Authenticator/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { Authenticator as AuthenticatorComponent } from '@aws-amplify/ui-react';
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+
+export class Authenticator {
+  constructor(container: HTMLElement) {
+    ReactDOM.render(
+      <AuthenticatorComponent>{() => <h1>Howdy</h1>}</AuthenticatorComponent>,
+      container
+    );
+  }
+}
```

Comment:
This is a great start! I am planning to make it more generic post this change gets merged.

Reply:
I'm curious what the behavior should be render props for an imperative wrapper 🤔 

Maybe:

```js
new Authenticator(container).show().then(({ session, signOut, user }) => {
  alert(`Howdy ${user.username}`)
});
```

Honestly, the "Vanilla Authenticator" example is a bit contrived – I don't have much customer data on what the use-cases are for this specifically.

Reply:
I think that's pretty much the correct analogy. Treating the return value from the imperative wrapper as a `ref` and acting accordingly would perhaps be the right usage.